packages/components/src/components/mobile-full-page-modal/mobile-full-page-modal.jsx

```diff
@@ -18,7 +18,9 @@ const MobileFullPageModalBodyWrapper = ({ className, children, should_wrap_body
 const MobileFullPageModal = ({
     body_className,
     className,
+    do_not_show_header_border,
```

Comment:
Can we change this to `show_header_border`? we can also set default value to `true`. 
Later you can change line 57 to truthy value as 
'dc-mobile-full-page-modal__header--border-bottom': show_header_border,
Throughout the code we can pass `show_header_border={false}` which makes it more readable.

Reply:
@matin-binary I changed the name in a way it has a positive meaning.